### PR TITLE
Add prow jobs for csi resizer

### PR DESCRIPTION
**Pull request by msau42**
Only create jobs against 1.15 for now.

### CODE REVIEW

**Comment by msau42** [comment]
```
ERROR: /home/prow/go/src/github.com/kubernetes/test-infra/config/tests/testgrids/BUILD.bazel:24:1: Executing genrule //config/tests/testgrids:testgrid_config failed (Exit 1)
2019/08/20 22:42:28 FAIL: could not apply prowjob annotations: no testgroup exists for job "pull-kubernetes-csi-external-resizer", but annotation "testgrid-tab-name" implies one should exist
```
/assign @chases2 
could you help look into this error?

File: config/jobs/kubernetes-csi/external-resizer/external-resizer-config.yaml [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    optional: false
+    decorate: true
+    skip_report: false
+    skip_branches: []
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    annotations:
+      test-grid-dsashboards: sig-storage-csi
```
Comment (chases2) [inline config/jobs/kubernetes-csi/external-resizer/external-resizer-config.yaml]:
```suggestion
      test-grid-dashboards: sig-storage-csi
```

Comment (msau42) [reply config/jobs/kubernetes-csi/external-resizer/external-resizer-config.yaml]: Good catch, I've updated the script and regenerated all the configs

**Pull request merged**